### ISSUE TITLE
Add a shorthand to use the underlying Prometheus's "default_registry" registry.

### ISSUE THREAD

**Issue by ajmwagar** [sd2k/rocket_prometheus#12]
```rust
 pub fn setup_metrics() -> PrometheusMetrics {
     let default_registry = prometheus::default_registry(); ‣default_registry: &Registry
     PrometheusMetrics::with_registry(default_registry.clone())
 }
```

Would you be willing to merge this upstream? or create an associated function doing the same?

**Comment by sd2k** [comment]
@ajmwagar Sounds reasonable to me, how does the linked PR look to you?

**Comment by ajmwagar** [comment]
Looks great! It's nice because it will auto-pick up the metrics from other sources.

**Comment by sd2k** [comment]
That's available in version 0.8.0, which has also been merged into the `async` branch if you're using that instead.
